### ISSUE TITLE
Resolver problem with `MiniCssExtractPlugin.loader`

### ISSUE THREAD

**Issue by ElPrudi** [webpro/knip#56]
**Knip version**: 1.9.0

It seems that the resolver has a problem with `MiniCssExtractPlugin.loader` as it is nested inside `MiniCssExtractPlugin`. Knip finds it under the name `<root>\node_modules\mini-css-extract-plugin\dist\loader.js` of which it cannot find a npm package.

### Debug Message
```
[
  '<root>\\node_modules\\mini-css-extract-plugin\\dist\\loader.js',
  'babel-loader',
  'base64-inline-loader',
  'css-loader',
  'less-loader',
  'ts-loader',
  'webpack-cli',
  'webpack-dev-server'
]
```
```
Unused dev dependencies (26)
...
mini-css-extract-plugin               package.json
...
```
```
Unlisted or unresolved dependencies (4)
...
<root>\node_modules\mini-css-extract-plugin\dist\loader.js  webpack.config.js
```
Either the plugin or the loader or both should be counted as using the package.

**Comment by webpro** [comment]
This should be fixed in [v1.10.0](https://github.com/webpro/knip/releases/tag/1.10.0)

**Comment by ElPrudi** [comment]
Can confirm. I don't have any unresolved dependencies anymore.
